impl/pom.xml

```diff
@@ -400,8 +420,12 @@
                         <manifestFile>${project.build.outputDirectory}/META-INF/MANIFEST.MF</manifestFile>
                     </archive>
                     <excludes>
-                        <exclude>**/*.java</exclude>
-                        <exclude>**/*.html</exclude>
+                        <exclude>jakarta/**</exclude>
+                        <exclude>com/sun/faces/vendor/GlassFish*</exclude>
+                        <exclude>com/sun/faces/vendor/Jetty*</exclude>
+                        <exclude>com/sun/faces/vendor/Tomcat*</exclude>
```

Comment:
Might be good to check if the above three files still exist in the `impl/src/main/java/com/sun/faces/vendor/` directory.